public/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
```

Comment:
Ooh very fancy with the code formatting, I see you! I think most folks prefer two space indentation for HTML, although I can't tell if those are actually true tabs because I'm on mobile; that's fine if that is the case.